ssl/quic/quic_record_rx.c

```diff
@@ -678,13 +740,14 @@ static ossl_inline void ignore_res(int x)
     /* No-op. */
 }
 
-static void qrx_key_update_initiated(OSSL_QRX *qrx)
+static void qrx_key_update_initiated(OSSL_QRX *qrx, QUIC_PN pn)
 {
     if (!ossl_qrl_enc_level_set_key_update(&qrx->el_set, QUIC_ENC_LEVEL_1RTT))
+        /* Returns 0 if already in RXKU, so we don't call callback again. */
         return;
```

Comment:
I assume `returns 0` in the comment refers to the ossl_qrl_enc_level_set_key_update() function return. Would it make sense to change the comment to : `We are already in RXKU, so we don't call callback again.`